agent/intentions_endpoint_test.go

```diff
@@ -62,6 +62,35 @@ func TestIntentionList(t *testing.T) {
 			ids = append(ids, reply)
 		}
 
+		// TODO(peering): when we handle Upserts, we can use the for loop above. But it may be that we
+		// rip out legacy intentions before supporting that use case so run a config entry request instead here.
+		testutil.RunStep(t, "set up peered intention", func(t *testing.T) {
```

Comment:
this shouldn't be in a subtest; it's setup code that belongs in the same level as the writes above. You could alternatively wrap it in braces to distinguish the setup from the actual assertion tests
```
// SETUP
{
...
}
```

Reply:
✅